content/blog/ticket-transfer-and-cancellation-2017.md

```diff
@@ -0,0 +1,12 @@
+Title: Ticket transfer and cancellation 2017
+Date: 2017-10-14 9:15
+Category: 2017
+Slug: ticket-transfer-&-cancel-2017
+Summary: Ticket transfer and Cancellation guidelines for Pycon India 2017
+
+PyCon India, the premier conference in India on using and developing the Python programming language is conducted annually by the Python developer community. The Ticket is for the main Conference on 4th and 5th Nov 2017. For DevSprints on 2nd and 3rd Nov 2017, one should have any one of the conference tickets. Tickets for workshops are separate and would be available once the schedule is finalized.
+
+The policy for ticket transfer & cancellation guidelines for the event is as follows :-
+All tickets bought are transferable. The early bird tickets cannot be canceled bought using the explara.com, those tickets will only be transferred. All other tickets can be canceled or transferred till 15th Oct 2017. For queries on ticketing or ticket cancellations, please write to [tickets@in.pycon.org](tickets@in.pycon.org) or check the [FAQs](https://in.pycon.org/2017/faqs.html).
+
+To transfer your PyCon India ticket or get a ticket transferred to you or your friend. Fill up the form at [http://bit.ly/pyind17transferform](http://bit.ly/pyind17transferform). Your entry will be added to [http://bit.ly/pyind17transfersheet](http://bit.ly/pyind17transfersheet). If you are contacted by anyone, visit [https://www.explara.com/manage-tickets](https://www.explara.com/manage-tickets) and fill the email id used at the time of booking and submit. Click on Transfer Ticket and follow the instructions to fill new attendee details. After transferring edit your form response and set status to 'Resolved' through the edit response link sent to your email. For people looking to get a ticket transferred to them, you may want to keep an eye on the sheet regularly.
```

Comment:
Instead of writing links you can change the sentence like "Fill up this [form](http://bit.ly/pyind17transferform)"
Same goes for all the other links. It'll look better.

Reply:
I should be "Dev Sprints and Workshops on 2nd and 3rd November"

Reply:
True that. Will implement.